docs/notebooks/utils/uploading_benchmarking.ipynb

```diff
@@ -14,7 +30,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Uploading Benchmarking Images from local data locations .\n",
+    "### Uploading Benchmarking Images from local data locations .\n",
```

Comment:
I think you should change the "define variables" section below in a couple ways
- change the description for "source," the data the benchmarking set is not in an [octree](https://en.wikipedia.org/wiki/Octree#:~:text=An%20octree%20is%20a%20tree,three%2Ddimensional%20analog%20of%20quadtrees.), so I think any mention of an octree is miselading.
- the description of p is a little incomplete, which is appropriate, but we should tell the user where they could find more information. They could find more information in the [cloudvolume](https://github.com/seung-lab/cloud-volume) package.
- nun_res is now irrelevant I believe, since this data only comes in one resolution